gtsam/discrete/DiscreteFactor.h

```diff
@@ -89,9 +89,20 @@ class GTSAM_EXPORT DiscreteFactor: public Factor {
   /// @name Wrapper support
   /// @{
   
-  /// Render as markdown table.
+  using Names = std::map<Key, std::vector<std::string>>;
+
+  static std::string Translate(const Names& names, Key key, size_t index);
```

Comment:
Documentation? Should be an easy one-liner.

Reply:
Done